frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.tsx

```diff
@@ -117,21 +130,29 @@ export const PipelineVisualizationTask: React.FC<PipelineVisualizationTaskProp>
 };
 const TaskComponent: React.FC<TaskProps> = ({ task, status, name, isPipelineRun }) => {
   const taskData = task.data;
+  const stepList = (taskData.spec && taskData.spec.steps) || [];
+  const inVisualMode = isPipelineRun && status && status.reason;
```

Comment:
What is this variable exactly storing?

Reply:
A boolean, it was a rendered state - I'll give it a type now that I see the confusion point.

Reply:
I'll rename the variable to `showStatusState` as well to indicate it's a flag to show the state of the status (ie, we are in the pipelinerun and want to see how the current run is doing)